cstrml/swork/src/lib.rs

```diff
@@ -489,27 +509,24 @@ decl_module! {
             // 2. Ensure who didn't join group right now
             ensure!(identity.group.is_none(), Error::<T>::AlreadyJoint);
```

Comment:
Make sure owner cannot be member and add corresponding test

Reply:
Done